stackbuilder.md

```diff
@@ -0,0 +1,28 @@
+# Restaurant Truffels in Haarlem 
+================================
+
+http://www.restauranttruffels.nl/
+
+A great restaurant in Haarlem, near Amsterdam.
+
+
+Lange Veerstraat 45
+2011 DA Haarlem
+
+
+On the menu:
```

Comment:
You may want to edit this file to include `-` (hyphens) so that these appear as a bulleted list!